Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -81,6 +81,12 @@ - (BOOL)application:(UIApplication *)application
         [self handleDeepLinkObject:universalObject linkProperties:linkProperties error:error];
     }];
 
+    
+    BranchEvent *earlyEvent = [BranchEvent standardEvent:BNCAddToCartEvent];
+    NSLog(@"Logging Early Event: %@", earlyEvent);
+    [earlyEvent logEvent];
```

Comment:
Is this supposed to queue an event before the sdk is initialized?

Reply:
Yeah